### PR TITLE
DOC: add section on groupby().rolling/expanding/resample

### LINKED ISSUE #14759: DOC: document groupby.resample/rolling
examples are in the whatsnew: http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#groupby-syntax-with-window-and-resample-operations

but I think a section in groupby.rst with some xrefs in timeseries.rst (resample) and computation.rst (rolling) would be helpful. I wrote this and can barely find it!

plus the API reference is missing these functions

### PR DESCRIPTION
**Pull request by wandersoncferreira**
- [x] closes #14759 

This is my first PR here. Not sure about the missing entries in API reference. Need some guidance on that. The same doubt about the best place to put the xref in computation.rst and timeseries.rst

Thanks.

### CODE REVIEW

**Comment by wandersoncferreira** [comment]
@jorisvandenbossche can you help me with more information about the failed test above?

**Comment by jorisvandenbossche** [comment]
@wandersoncferreira you can ignore the AppVeyor error, that is a problem on our end unrelated to this PR. Travis did pass.

File: doc/source/computation.rst [hunk 1]
```diff
@@ -214,6 +214,10 @@ computing common *window* or *rolling* statistics. Among these are count, sum,
 mean, median, correlation, variance, covariance, standard deviation, skewness,
 and kurtosis.
 
+Now the ``rolling()`` and ``expanding()`` functions can be used directly from
```
Comment (jreback) [inline doc/source/computation.rst]: say starting in 0.18.1

File: doc/source/groupby.rst [hunk 2]
```diff
@@ -614,6 +614,73 @@ and that the transformed data contains no NAs.
...
+examples of the application of each of the refered methods:
+
+
+Example of the ``rolling()`` method applied to groupbys:
+
+.. ipython:: python
+
+   df = pd.DataFrame({'A': [1] * 20 + [2] * 12,
+	              'B': np.arange(32)})
+
```
Comment (jreback) [inline doc/source/groupby.rst]: add a line showing df

File: doc/source/groupby.rst [hunk 3]
```diff
@@ -614,6 +614,73 @@ and that the transformed data contains no NAs.
...
+   df.groupby('A').rolling(4).B.mean()
+
+.. note::
```
Comment (jreback) [inline doc/source/groupby.rst]: you don't need the note

File: doc/source/groupby.rst [hunk 4]
```diff
@@ -614,6 +614,73 @@ and that the transformed data contains no NAs.
...
+
+   df = pd.DataFrame({'date': pd.date_range(start='2016-01-01',
+	                                    periods=4,
+					    freq='W'),
+		      'group': [1, 1, 2, 2],
+		      'val': [5, 6, 7, 8]}).set_index('date')
+
+   df.groupby('group').resample('1D').ffill()
+
+.. note::
```
Comment (jreback) [inline doc/source/groupby.rst]: remove the note, show df (as its different from above).

File: doc/source/timeseries.rst [hunk 5]
```diff
@@ -1288,6 +1288,9 @@ limited to, financial applications.
 ``.resample()`` is a time-based groupby, followed by a reduction method on each of its groups.
 See some :ref:`cookbook examples <cookbook.resample>` for some advanced strategies
```
Comment (jreback) [inline doc/source/timeseries.rst]: starting in 0.18.1

**Comment by wandersoncferreira** [comment]
@jreback thanks for your comments. Let me know if I can make it even better.

**Comment by jorisvandenbossche** [comment]
@wandersoncferreira Thanks!

**Pull request merged**